tests/semantic_analyser.cpp

```diff
@@ -698,6 +702,31 @@ TEST(semantic_analyser, call_ntop)
   test(structs + "kprobe:f { ntop(((struct inet*)0)->invalid); }", 1);
 }
 
+TEST(semantic_analyser, call_pton)
```

Comment:
Please add some tests that fail. For example invalid arg types. And non-literal strings

Reply:
And also some tests for invalid addresses.